docs/using-the-ushahidi-platform-v3-v4/untitled.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
I think this file can be discarded . There is documentation for the v3 to v4 upgrade in master already: https://github.com/ushahidi/platform/tree/master/docs/upgrading-ushahidi  .

I'm not 100% sure if this document was intending to explain anything else...